generators/reference-set-generator.py

```diff
@@ -106,6 +106,7 @@ def generateCDef2(iterationsMap, iteration, t):
 def generateCDef3(iterationsMap, iteration, t):
   msg = generateGenericMessage("EiffelCompositionDefinedEvent", t, "1.0", "CDef3", iteration)
   linka(msg, findLatestPrevious(iterationsMap, iteration, "CDef3"), "previousVersions")
+  linka(msg, findLatestPrevious(iterationsMap, iteration + 1, "SCS1"), "elements")
```

Comment:
It was just a copy-paste, I didn't think of it that much =) I will change it to dig it out from iterationsMap directly.